api/v1beta1/common_types.go

```diff
@@ -54,3 +56,32 @@ type CinderServiceDebug struct {
 	// service enable debug
 	Service bool `json:"service,omitempty"`
 }
+
+// MetalLBConfig to configure the MetalLB loadbalancer service
+type MetalLBConfig struct {
```

Comment:
I made a similar comment in the glance-operator patch and I'm wondering if we want to have a single definition of this MetalLB struct in lib-common instead of replicating it across the operators

Reply:
we plan to move this into the infra-operator when we add there more network related stuff, like ipam, dns and so to have it versioned there within the api. until then we keep it in the operators

Reply:
Thanks, let's move to the `infra-operator` and import the corresponding struct in a follow up PR.